priam/src/main/java/com/netflix/priam/scheduler/PriamScheduler.java

```diff
@@ -29,15 +29,15 @@
 public class PriamScheduler {
     private static final Logger logger = LoggerFactory.getLogger(PriamScheduler.class);
     private final Scheduler scheduler;
-    private final GuiceJobFactory jobFactory;
+    private final PriamJobFactory jobProvider;
```

Comment:
Let's keep terms consistent. That is, use jobFactory rather than jobProvider. I know we're moving away from Guice, but Provider has a specific meaning in that context.